source/plugin/Assets/Plugins/iOS/GADUServerSideVerificationOptions.h

```diff
@@ -0,0 +1,22 @@
+//
+//  GADServerSideVerificationOptions.h
+//  Google Mobile Ads SDK
+//
+//  Copyright 2018 Google Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <GoogleMobileAds/GoogleMobileAds.h>
+
+/// Options for server-to-server verification callbacks for a rewarded ad.
+@interface GADUServerSideVerificationOptions : NSObject
```

Comment:
is this class adding anything that isn't already available from the `GADServerSideVerificationOptions` class?  It looks a bit redundant to me. Why not remove it?

Reply:
Done. I have added that as a wrapper class, it is not required.